cmsis/TARGET_CORTEX_M/mbed_fault_handler.c

```diff
@@ -20,17 +20,23 @@
 #include <inttypes.h>
 
 #include "device.h"
-#include "platform/mbed_error.h"
-#include "platform/mbed_interface.h"
+#include "mbed_error.h"
+#include "mbed_interface.h"
+#include "mbed_crash_data_offsets.h"
 
 #ifndef MBED_FAULT_HANDLER_DISABLED
 #include "mbed_fault_handler.h"
 
 //Functions Prototypes
 void print_context_info(void);
 
-//Global for populating the context in exception handler
-mbed_fault_context_t mbed_fault_context;
+#if MBED_CONF_PLATFORM_CRASH_CAPTURE_ENABLED
+    //Global for populating the context in exception handler
+    mbed_fault_context_t *mbed_fault_context=(mbed_fault_context_t *)((uint32_t)FAULT_CONTEXT_LOCATION);
```

Comment:
Pointers should be const to save RAM (`mbed_fault_context * const mbed_fault_context`)

Reply:
Fixed